include/range/v3/action/drop.hpp

```diff
@@ -34,20 +36,22 @@ namespace ranges
         private:
             friend action_access;
             template<typename Int>
-            static auto CPP_fun(bind)(drop_fn drop, Int n)(
+            static auto CPP_fun(bind)(drop_fn drop, Int n)( //
```

Comment:
I really have no idea what is the clang-format's reason for a trailing `//` before every `requires`. :slightly_smiling_face:

Reply:
clang-format didn't add that; I did, so that clang-format wouldn't make a hash of the formatting. It has no idea what a _requires-clause_ is.